mmv1/products/vertexai/IndexEndpoint.yaml

```diff
@@ -48,11 +48,20 @@ examples:
   - !ruby/object:Provider::Terraform::Examples
     name: "vertex_ai_index_endpoint"
     primary_resource_id: "index_endpoint"
+    skip_docs: true
```

Comment:
we'd prefer to keep example of this config in the test but skip `vertex_ai_index_endpoint_with_false_psc` instead, since `vertex_ai_index_endpoint` is a standard basic config for the resource

Reply:
@shuyama1 Thank you for catching this mistake 🙏 It was unintentional. I moved `skip_docs: true` to the example: [8d0230b](https://github.com/GoogleCloudPlatform/magic-modules/pull/8851/commits/8d0230bffc6ed7592331225c441a148cd58c6c05).